src/core/ColorPicker/ColorPalette.test.tsx

```diff
@@ -15,7 +15,12 @@ beforeAll(() => {
 });
 
 it('should render in its most basic state', () => {
-  const colors = ['#9ba5af', '#23450b', '#00121d', '#002a44'];
+  const colors = [
```

Comment:
why changed?

Reply:
Color swatch now uses hslString so the --swatch-color check was failing in these tests